pages/build-promotion.md

```diff
@@ -1,10 +1,10 @@
 ---
 permalink: /build-promotion/
 layout: default
-title: Build Promotion
+title: Build promotion
 ---
 <a name="build-promotion"></a>
-* Every code commit to version control creates a potentially deployable artifact in the Continuous integration server.
-Therefore a well defined and version controlled process for promoting a build from Dev(or continuous integration artifact)->Test->Staging->Prod needs to exist.
-* These built artifacts should not change when deploying to multiple environments(i.e. built once), and are stored in a artifact repository for easy retrieval. 
-This ensures that the built product is the same when testing in each environment, and that only configuration changes(if any) are different between environments.
+* Every code commit to version control creates a potentially deployable artifact in the continuous integration server.
+This being the case, we need a well defined and version controlled process for promoting a build (a continuous integration artifact) from development. ->Test->Staging->Prod.
```

Comment:
And we have a quorum! Made the change. :)

On Thu, Aug 6, 2015 at 3:11 PM, John Scancella <notifications@github.com>
wrote:

> In pages/build-promotion.md
> <https://github.com/18F/dev-environment/pull/8#discussion_r36453406>:
>
> >  ---
> >  <a name="build-promotion"></a>
> > -* Every code commit to version control creates a potentially deployable artifact in the Continuous integration server.
> > -Therefore a well defined and version controlled process for promoting a build from Dev(or continuous integration artifact)->Test->Staging->Prod needs to exist.
> > -* These built artifacts should not change when deploying to multiple environments(i.e. built once), and are stored in a artifact repository for easy retrieval.
> > -This ensures that the built product is the same when testing in each environment, and that only configuration changes(if any) are different between environments.
> > +* Every code commit to version control creates a potentially deployable artifact in the continuous integration server.
> > +This being the case, we need a well defined and version controlled process for promoting a build (a continuous integration artifact) from development. ->Test->Staging->Prod.
>
> Agree with @mbland <https://github.com/mbland> "from development to test,
> staging, and eventually production." sounds good
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/18F/dev-environment/pull/8/files#r36453406>.
>